### DISCUSSION TITLE
are the include and exclude buttons functional in Pre-processing table view?

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#52] [Q&A] [answered]
I would like to manually discard some tilt series by clicking exclude in the Table view.  However, when I apply filters, the total number is unchanged. Also there is no indication in the GUI that a line was deselected in the table. I would be nice to highlight the removed micrographs in the list.

**Comment by abartesaghi** [comment, marked as answer]
See [here](https://nextpyp.app/files/pyp/latest/docs/guide/filters.html) for a description of how filters work. We will consider highlighting rows for excluded micrographs/tilt-series in the future.

**Reply by JuhaHuiskonen** [reply]
Thanks!  The last few columns of the table were not visible on my laptop so I had missed the "exclude" column
